library/psa_crypto_core.h

```diff
@@ -68,7 +68,7 @@ typedef struct
 
 /* A mask of key attribute flags used only internally.
  * Currently there aren't any. */
-#define MBEDTLS_PSA_KA_MASK_INTERNAL_ONLY (     \
+#define PSA_KA_MASK_INTERNAL_ONLY (     \
```

Comment:
Is there any policy for PSA macros similar to this one with MbedTLS ones?

Reply:
No. Mbed TLS had no internal headers (`library/*.h`), but Mbed Crypto does in its PSA parts (`library/psa_*.h`).